packages/clay-management-toolbar/src/ActiveState.tsx

```diff
@@ -0,0 +1,252 @@
+								])}
+								<span className="navbar-breakpoint-down-d-none">
+									{labels.selected}
+								</span>
+							</>
+						)}
+					</span>
+				</Item>
+
+				{supportsBulkActions && (
+					<Item className="nav-item-shrink">
+						{clearSelectionURL ? (
+							<ClayLink
+								className="nav-link"
+								href={clearSelectionURL}
+							>
+								<span className="text-truncate-inline">
+									<span className="text-truncate">
+										{labels.clearSelection}
+									</span>
+								</span>
+							</ClayLink>
+						) : (
+							<ClayButton
+								className="nav-link"
+								displayType="unstyled"
+								onClick={onClearSelectionClick}
+								type="button"
+							>
+								<span className="text-truncate-inline">
+									<span className="text-truncate">
+										{labels.clearSelection}
+									</span>
+								</span>
+							</ClayButton>
+						)}
+					</Item>
+				)}
+
+				{showSelectAllButton && (
+					<Item className="nav-item-shrink">
+						{selectAllURL ? (
+							<ClayLink className="nav-link" href={selectAllURL}>
+								<span className="text-truncate-inline">
+									<span className="text-truncate">
+										{labels.selectAll}
+									</span>
+								</span>
+							</ClayLink>
+						) : (
+							<ClayButton
+								className="nav-link"
+								displayType="unstyled"
+								onClick={onSelectAllClick}
+								type="button"
+							>
+								<span className="text-truncate-inline">
+									<span className="text-truncate">
+										{labels.selectAll}
+									</span>
+								</span>
+							</ClayButton>
+						)}
+					</Item>
+				)}
+			</ItemList>
+
+			{(actionItems.length > 0 || showInfoButton) && (
+				<>
+					<ItemList>
+						{showInfoButton && (
+							<Item>
+								<ClayButton
+									className="nav-link nav-link-monospaced"
+									displayType="unstyled"
+									onClick={onInfoButtonClick}
+									title={titlesLang.infoButton}
+								>
+									<ClayIcon
+										spritemap={spritemap}
+										symbol="info-circle-open"
+									/>
+								</ClayButton>
+							</Item>
+						)}
+					</ItemList>
+
+					{actionItems.length > 0 && (
+						<>
+							{actionItems.map(
+								({quickAction, ...item}, index) =>
+									quickAction &&
+									item.symbolLeft && (
+										<Item
+											className="navbar-breakpoint-down-d-none"
+											key={index}
+										>
+											{item.href && !item.disabled ? (
+												<ClayLink
+													className="nav-link nav-link-monospaced"
+													href={item.href}
+													title={item.label}
+												>
+													<ClayIcon
+														spritemap={spritemap}
+														symbol={item.symbolLeft}
+													/>
+												</ClayLink>
+											) : (
+												<ClayButton
+													className="nav-link nav-link-monospaced"
+													disabled={item.disabled}
+													displayType="unstyled"
+													onClick={onQuickActionClick}
```

Comment:
Won't each actionItem have a different onClick?

Reply:
Yes you can, I think I got it from Soy's use cases. There you have only one event that returns the element you clicked. I think we can do the same here:

```
onClick={event => onQuickActionClick(event, item)}
```